### PR TITLE
Worker Identity

### PR DESCRIPTION
**Pull request by ssz1997**
Create a default directory on host machine and mount into worker pods. Worker will then write its identity file into this directory so that it can be persisted in k8s env.

### CODE REVIEW

**Comment by ssz1997** [comment]
@lucyge2022 This PR persists worker id file to host machine, but not including securing the file (chmod 400) because that has to be done on the operator side - only helm can't do it. Let's get this one in first.
@JiamingMai PTAL. Thanks!

File: deploy/charts/alluxio/templates/helpers/_alluxio-site-properties.tpl [hunk 1]
```diff
@@ -46,6 +46,9 @@ alluxio.worker.page.store.type=LOCAL
 alluxio.worker.page.store.dirs={{ .Values.pagestore.hostPath }}
 {{ printf "alluxio.worker.page.store.sizes=%v" .Values.pagestore.quota }}
 
+# Worker Identity
+{{ printf "alluxio.worker.identity.uuid.file.path=%v/worker_identity" (include "alluxio.mount.basePath" "/worker_identity") }}
```
Comment (lucyge2022) [inline deploy/charts/alluxio/templates/helpers/_alluxio-site-properties.tpl]: i thought %v maps to the "alluxio.mount.basePath" what's the second "/worker_identity" following after?

File: tests/helm/expectedTemplates/conf/configmap.yaml [hunk 2]
```diff
@@ -45,6 +45,9 @@ data:
     alluxio.worker.page.store.type=LOCAL
     alluxio.worker.page.store.dirs=/mnt/alluxio/pagestore
     alluxio.worker.page.store.sizes=1Gi
+    
+    # Worker Identity
+    alluxio.worker.identity.uuid.file.path=/mnt/alluxio/worker_identity/worker_identity
```
Comment (lucyge2022) [inline tests/helm/expectedTemplates/conf/configmap.yaml]: ? why two worker_identity/?

File: tests/helm/expectedTemplates/worker/deployment.yaml [hunk 3]
```diff
@@ -192,6 +192,8 @@ spec:
           volumeMounts:
             - name: dummy-alluxio-alluxio-conf
               mountPath: /opt/alluxio/conf
+            - name: dummy-alluxio-worker-identity-volume
+              mountPath: /mnt/alluxio/worker_identity
```
Comment (lucyge2022) [inline tests/helm/expectedTemplates/worker/deployment.yaml]: use /mnt/alluxio/systeminfo  as discussed offline.

File: tests/helm/expectedTemplates/conf/configmap.yaml (hunk 2 above)
Comment (lucyge2022) [reply tests/helm/expectedTemplates/conf/configmap.yaml]: this becomes /mnt/alluxio/systeminfo/worker_identity  as discussed offline.

**Comment by ssz1997** [comment]
@lucyge2022 PTAL. Thanks!

**Review by lucyge2022 — APPROVED** [review]
LGTM.

**Pull request merged**